connector-sdk/core/src/main/java/io/camunda/connector/api/inbound/Health.java

```diff
@@ -39,12 +59,18 @@ public Map<String, Object> getDetails() {
     return details;
   }
 
+  public HealthError getError() {
+    return error;
+  }
+
```

Comment:
I thought we return the error as part of the details?

Reply:
I thought we wanted to check the details for a property named `error`. If its there the modeler renders it as an error.

Reply:
@sbuettner based on our previous conversations my assumption was that we agreed on adding an error construct.
Architecture session:
![image](https://github.com/camunda/connectors/assets/119574841/68bce602-1f8b-4d71-9669-932c3faf8fa9)
Slack:
![image](https://github.com/camunda/connectors/assets/119574841/52a455c2-2ec9-4f5c-87b8-82d093a23055)
WDYT?

Reply:
But maybe I misunderstood it, and it is enough to add this to the details. My issue with that is we then need to be able to effectively remove it, but if we accept that the "error" is a reserved keyword, than we can certainly do that. This way it makes sense, I will add it to the details with the key "error". Sorry for the confusion.